src/components/mapping/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from "@emotion/styled"
+import { Popup } from "react-map-gl"
+import "mapbox-gl/dist/mapbox-gl.css"
+
+import { Card } from "../../cards/Card/Card"
+import { Coordinate } from "../types/types"
+
+interface TooltipProps {
+  label: JSX.Element | null
+  position: Coordinate
+}
+
+export const MapTooltip = ({ label, position }: TooltipProps): JSX.Element | null => {
```

Comment:
Pretty sure `null` is a valid `JSX.Element`:
```suggestion
export const MapTooltip = ({ label, position }: TooltipProps): JSX.Element => {
```

Reply:
I get `Type 'null' is not assignable to type 'Element'.` 🤔